MvvmCross/Binding/Droid/Target/MvxSearchViewQueryTextTargetBinding.cs

```diff
@@ -29,6 +29,8 @@ public override void SubscribeToEvents()
 
         protected override void SetValueImpl(object target, object value)
         {
+            if (target is SearchView sv)
```

Comment:
It's already guaranteed to be a `SearchView` and it can't be null (the caller checks that it is not and that the Java handle is valid). Note that I'm on mobile if this sounds terse.